web-client/src/components/training/TrainingInformation.vue

```diff
@@ -180,6 +226,44 @@ const accuracySeries = computed(() => props.logs
     validation: [] as number[],
   })
 )
+const lossSeries = computed(() => props.logs
+  .flatMap((round) =>
+    round.epochs.map((epoch) => { return {
+      training: epoch.training.loss,
+      validation: (epoch.validation?.loss ?? 0),
+    }})
+  )
+  .reduce(({ training, validation }, cur) => { return {
+    training: training.concat([cur.training]),
+    validation: validation.concat([cur.validation]),
+  }}, {
+    training: [] as number[],
+    validation: [] as number[],
+  })
+)
+
+const yAxisMax = computed(() => {
+  let maxVal = Math.max(...[...lossSeries.value.training, ...lossSeries.value.validation])
+  return (maxVal > 0) ? maxVal : 10 // if Math.max returns -inf or 0, set the max to 10 arbitrarily
+})
+
+const lossChartsOptions = computed(() => {
+  return {
+    ...options,
+    ...{
+      yaxis: {
+        max: () => yAxisMax.value,
+        min: 0,
+        labels: {
+          show: true,
+          formatter: function (value: number) {
+            return value.toFixed(2);
+          }
+        }
+      }
+    }
```

Comment:
no need for double spread

```suggestion
    yaxis: {
      max: () => yAxisMax.value,
      min: 0,
      labels: {
        show: true,
        formatter: function (value: number) {
          return value.toFixed(2);
        }
      }
    }
```